### PR TITLE
Part - 6

### PR DESCRIPTION
**Pull request by dariiabutorina**
- **_Order_** : model, dao, service created
- **_hibernate.cfg.xml_** : mapping added
- **_ShoppingCartService_** : method clear() modified

### CODE REVIEW

File: src/main/java/com/dev/cinema/service/implementation/ShoppingCartServiceImpl.java [hunk 1]
```diff
@@ -53,7 +54,7 @@ public void addSession(MovieSession movieSession, User user) {
 
     @Override
     public void clear(ShoppingCart shoppingCart) {
-        shoppingCart.getTickets().clear();
+        shoppingCart.setTickets(Collections.emptyList());
```
Comment (andromash) [inline src/main/java/com/dev/cinema/service/implementation/ShoppingCartServiceImpl.java]: Collections.emptyList() will return immutable list. Is this okay for our purpose?

**Review by andromash — CHANGES_REQUESTED** [review]
Nice!

File: src/main/java/com/dev/cinema/service/implementation/ShoppingCartServiceImpl.java (hunk 1 above)
Comment (dariiabutorina) [reply src/main/java/com/dev/cinema/service/implementation/ShoppingCartServiceImpl.java]: Yes, it works. Tested

Comment (dmytro-hryhoruk) [reply src/main/java/com/dev/cinema/service/implementation/ShoppingCartServiceImpl.java]: That's okay cause later you are fetching it from DB

**Review by dmytro-hryhoruk — APPROVED** [review]
👍

**Pull request merged**